cli.js

```diff
@@ -170,6 +171,56 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   }, async (argv) => {
     await handleCommand(argv, listHandler);
   })
+  .command('triggers', 'Manages triggers', yargs0 => yargs0
+      .demandCommand()
+      .command('upsert', 'Creates a new trigger or updates an existing one', (yargs1) => {
+        yargs1
+          .usage('Usage: $0 triggers upsert <trigger name> [options]')
+          .positional('trigger name', {
+            describe: 'Trigger name',
+            type: 'string',
+          })
+          .option('source-uri', {
+            demandOption: true,
+            describe: 'Redis-stream uri',
+            type: 'string',
+          })
+          .option('target-func', {
+            demandOption: true,
+            describe: 'Name of the function to run for each item in the stream',
+            type: 'string',
+          })
+          .option('error-handler-func', {
+            describe: 'Name of the function to run when an error occurres',
+            type: 'string',
+          })
+          .demand(1)
```

Comment:
`demand` is deprecated according to https://github.com/yargs/yargs/blob/master/docs/api.md#demand.
can this be replaced with `demandOption` or `demandCommand`?

Reply:
Changing all `demand` occurrences to use `demandCommand` instead